index.js

```diff
@@ -12,6 +17,14 @@ app.get('/', (_request, response) => {
   response.status(HTTP_OK_STATUS).send();
 });
 
+app.use('/talker', routes);
+
+app.post('/login', email, password, (_req, res) => {
+  // const token = crypto.randomBytes(8).toString('hex');
+  const token = createToken; 
+  res.status(200).json({ token: token() });
```

Comment:
O que acha de chamar a função createToken antes e só passar token pro .json()?

```suggestion
  const token = createToken(); 
  res.status(200).json({ token });
```
Dessa forma.